db/migrate/20170216193758_add_vm_table_comments.rb

```diff
@@ -0,0 +1,81 @@
+class AddVmTableComments < ActiveRecord::Migration[5.0]
+  def change
+    change_table_comment :vms, "Virtual Machines and Templates"
+
+    change_column_comment :vms, :autostart, "A boolean value that indicates whether or not the VM is set to autostart."
+    change_column_comment :vms, :availability_zone_id, "Availability zone that the VM is part of. FK to the availability_zones table."
```

Comment:
the "FK" statement is typically implemented as a foreign key reference.
statements like "for this record" "that this record is a part of" is just verbiage.

Is there actually anything we can tell a non-rails person?